rs/nervous_system/tools/submit-motion-proposal/BUILD.bazel

```diff
@@ -0,0 +1,27 @@
+load("@rules_rust//rust:defs.bzl", "rust_binary")
+
+package(default_visibility = ["//visibility:private"])
+
+DEPENDENCIES = [
+    # Keep sorted.
+    "//rs/nns/common",
+    "//rs/nns/constants",
+    "//rs/nns/governance/api",
+    "@crate_index//:candid",
+    "@crate_index//:clap",
+    "@crate_index//:ic-agent",
+    "@crate_index//:ic-certification",
+    "@crate_index//:ic-identity-hsm",
+    "@crate_index//:serde",
+    "@crate_index//:serde_json",
+    "@crate_index//:serde_yaml",
+    "@crate_index//:tokio",
+]
+
+rust_binary(
+    name = "submit-motion-proposal",
+    srcs = [
+        "src/main.rs",
+    ],
+    deps = DEPENDENCIES,
```

Comment:
As @nmattia explains in [this slack thread](https://dfinity.slack.com/archives/CH4CADCJX/p1753198380435869), we're moving towards inlining dependencies to reduce the chance of introducing unused dependencies and to simplify catching and fixing them. So I would suggest doing the same here.

Reply:
Done.